dkeycache/App/main.cpp

```diff
@@ -3,6 +3,8 @@
 #include <signal.h>
 
 #include <enclave_u.h>
+#include <getopt.h>
```

Comment:
change the commit title as 
“
pass the dkeyserver addr as input params
”

Reply:
changed